toolkit/imageconfigs/additionalconfigs/cloud-init.cfg

```diff
@@ -0,0 +1,120 @@
+# The top level settings are used as module
+# and system configuration.
+
+# A set of users which may be applied and/or used by various modules
+# when a 'default' entry is found it will reference the 'default_user'
+# from the distro configuration specified below
+users:
+   - default
+   - name: root
+     lock-passwd: false
+
+# If this is set, 'root' will not be able to ssh in and they 
+# will get a message to login instead as the above $user (ubuntu)
+disable_root: false
+
+#Vmware guest customization.
+disable_vmware_customization: true
+
+# This will cause the set+update hostname module to not operate (if true)
+preserve_hostname: false
+
+# datasources to check for cloud-config
+datasource_list: [ 
+                  NoCloud, 
+                  ConfigDrive,
+                  OVF,
+                  OpenStack,
+#                  OpenNebula, 
+#                  AltCloud, 
+#                  MAAS, 
+#                  GCE, 
+#                  CloudSigma, 
+#                  Ec2, 
+#                  CloudStack,
+#                  VmxGuestinfo, 
+                  None,
+                  Azure
```

Comment:
If we bump Azure to the top of the list here will it still work in the cloud and in Hyper-V?

Reply:
Done